package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "perf-goggles",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "git@github.com:csd713/perf-goggles.git",
+  "author": "Chiranjeevi <jeevi146@gmail.com>",
+  "license": "MIT",
```

Comment:
I checked perf-goggles and I don't know why I chose MIT.
Could you please remove this and instead add a LICENSE file like [this](https://github.com/mozilla/firefox-code-coverage-frontend/blob/master/LICENSE)?